js/main.js

```diff
@@ -25,4 +25,81 @@ Pineapples can be consumed fresh, cooked, juiced, or preserved. They are found i
 
 Strawberry
 The garden strawberry (or simply strawberry; Fragaria × ananassa) is a widely grown hybrid species of the genus Fragaria (collectively known as the strawberries). It is cultivated worldwide for its fruit. The fruit (which is not a botanical berry, but an aggregate accessory fruit) is widely appreciated for its characteristic aroma, bright red color, juicy texture, and sweetness. It is consumed in large quantities, either fresh or in such prepared foods as preserves, fruit juice, pies, ice creams, milkshakes, and chocolates. Artificial strawberry flavorings and aromas are also widely used in many products like lip gloss, candy, hand sanitizers, perfume, and many others.
-*/
\ No newline at end of file
+*/
+
+$(document).ready(function() {
+	function removeActive() {
```

Comment:
Nice use of a function. To simplify this, you could have changed the lines below to all be one of the following:

```js
$('li').removeClass('active');
```
or 
```js
$('.active').removeClass('active');
```